docs/terms/execution_engine.md

```diff
@@ -108,7 +108,7 @@ For specifics on the required keys for a given Execution Engine, please see our
     - a validation_graph object is ready. Nodes are MetricConfigurations, edges are dependencies.
 5. `_parse_validation_graph`
 6. for each set of ready_metrics: Execution Engine resolve_metrics
-7. for each metric: bundleable?
```

Comment:
This whole section still feels patchy to me, like it could use more narrative and explanation.

Reply:
I think that the document it was originally pulled from may have been some design notes that were pasted into the V2 docs and then copied over in the move to V3.  Since it also looks patchy to you, I've removed the section.  I thought it might contribute something to the understanding of the process that Execution Engines perform if read by someone more experienced than I was, but from what you're saying I don't think it's a significant contribution to a practical understanding of Execution Engines.